content/news/public-oss-demo/2023/upcoming-live-update-1-2023.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title:  "Announcing January PrestaShop Project Live Update"
+subtitle: "Let's meet for the first Live Update in 2023"
+date:   2023-01-06 09:00:00
+authors: [ kpodemski ]
+icon: icon-video-camera
+tags: [contribute, event, community]
+---
+
+Dear community,
+
+The first Live Update for the PrestaShop Project will happen on [**Wednesday, January 11th, 2023 at 4pm CET**](https://www.youtube.com/watch?v=YDS51o48W1I)!
+It was decided to switch from _Public Demo_ to _Live update_, as it would be great to experiment with formats other than you could see in previous sessions.
```

Comment:
Why not just say that the public demo is now called live update?

Reply:
I think it's worth explaining :) I've improved this part.